### PR TITLE
chore(canary): update canary to match expected Amplify npm versions

### PR DESCRIPTION
**Pull request by wlee221**
#### Description of changes

build-tests have been failing because `ui-react@v3-nex`t is not compatible with latest `aws-amplify`. This has been fixed on `main`, but not on `v3` branch.

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by 0618 — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

File: canary/apps/react/cra-ts/package.json [hunk 1]
```diff
@@ -3,15 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@aws-amplify/ui-react": "^2.2.1",
+    "@aws-amplify/ui-react": "latest",
```
Comment (zchenwei) [inline canary/apps/react/cra-ts/package.json:6]: Should this be v3 latest? It seems running against latest on v3 branch is meaningless

Comment (wlee221) [reply canary/apps/react/cra-ts/package.json:6]: Yep, updating.

**Review by reesscot — APPROVED** [review]

File: canary/apps/angular/angularcli/package.json [hunk 2]
```diff
@@ -20,8 +20,8 @@
     "@angular/platform-browser": "~11.2.14",
     "@angular/platform-browser-dynamic": "~11.2.14",
     "@angular/router": "~11.2.14",
-    "@aws-amplify/ui-angular": "latest",
-    "aws-amplify": "latest",
+    "@aws-amplify/ui-angular": "v3-latest",
```
Comment (wlee221) [inline canary/apps/angular/angularcli/package.json:23]: Updated to v3-latest. This isn't published yet, but will be hotfix is done. Note that CI will change to `v3-next` during build tests, so build tests aren't blocked.

**Review by zchenwei — APPROVED** [review]

**Comment by wlee221** [comment]
Will merge after I verify build-tests to succeed locally.

**Pull request merged**